src/lib/components/media/riven/item-manual-scrape.svelte

```
@@ -0,0 +1,867 @@
+                
+                // Custom Ranks
+                const categories = ["quality", "rips", "hdr", "audio", "extras", "trash"];
+                categories.forEach(cat => {
+                    const key = cat as keyof typeof ranking.custom_ranks;
+                    if (ranking.custom_ranks && ranking.custom_ranks[key]) {
+                        const categoryObj = ranking.custom_ranks[key];
+                        rankingOptions[cat] = Object.keys(categoryObj);
+                        
+                        // Populate selected options for this category
+                        newSelectedOptions[cat] = Object.entries(categoryObj)
+                            .filter(([_, val]) => {
```

ESLint

[error] '_' is defined but never used.

(@typescript-eslint/no-unused-vars)